scheduled-jobs/build/check-bugs/Jenkinsfile

```diff
@@ -4,11 +4,26 @@ properties( [
     disableResume(),
 ] )
 
-def commonlib = load("pipeline-scripts/commonlib.groovy")
-commonlib.ocpVersions.each { it ->
+def sortedVersions() {
+  return commonlib.ocpVersions.sort(false)
```

Comment:
This actually sorts versions by alphabetical order, which is technically incorrect.
You may correct the logic to sort versions, or don't sort at all.

Reply:
I removed the sorting. By the way, the function is used in a few places, see for example https://github.com/openshift/aos-cd-jobs/blob/master/scheduled-jobs/scanning/images-health/Jenkinsfile#L24

If you think that's incorrect, we might want to fix this across the whole repo